packages/qtbase.rb

```diff
@@ -3,23 +3,22 @@
 class Qtbase < Package
   description 'Qt Base (Core, Gui, Widgets, Network, ...)'
   homepage 'https://code.qt.io/cgit/qt/qtbase'
-  @_ver = '5.15.2'
-  version @_ver + '-1'
+  @_ver = 'kde-5.15'
```

Comment:
Qt should be independent from kde.  Not sure why that is included in the version.

Reply:
Its not dependent upon kde. This is just using the kde patched qtbase, since qt no longer releases patches publicly for the 5.15 series.

Not sure where that info should be put, but it seems kde should get credit for that...

Reply:
Fair enough.  Somebody might see that and think they need to install KDE dependencies.  That is the only issue I have with including kde in the version.

Reply:
Any suggestions on how you would like the versioning changed? Or should I leave it as is?

Reply:
It looks like invent.kde.org repo is using Gitlab so use the tagged branch as the version?

Reply:
done.